bluesky/plan_stubs.py

```diff
@@ -689,44 +689,58 @@ def configure(obj, *args, **kwargs):
     return (yield Msg('configure', obj, *args, **kwargs))
 
 
-def stage(obj):
+def stage(obj, *, group=None, wait=False):
```

Comment:
@tacaswell should this default to waiting for completion?

Reply:
I lean "no", for consistency with `abs_set` and others. These low-level plans should be "async" (in the general sense) by default.

Reply:
That's true, if we have an old-style stage then it will wait no matter what we pass here. Which begs the question, should we make the return from the `stage` message always be a `Status` object?

Reply:
Decision from call:

```py
ret = yield Msg("stage", obj, group=group)
if wait is None:
    # Wait for old-style devices (no choice!) but do not wait for new-style.
    wait = not isinstance(ret, Status)
if wait:
    yield from Msg("wait", None, group=group)
```